src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  margin-top: 17px;
```

Comment:
use BEM-mix to add external styles

in other words, you can add element stars to the product block in products.scss (but also keep stars as a separate block):
`product__stars`
It's allowed to set external styles (margin) using such selectors

then in HTML:

`<div class="stars product__stars stars--4">`

Reply:
styles for stars I need whrite in product.scss and for class product__stars ? and class stars I don't need it anymore ?